src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+
+import {
+  DEFAULT_PROPS,
+  STATS_ICONS,
+  STATS_LABELS,
+  DEFAULT_AVATAR_ICON,
+  FALLBACK_INITIALS,
+} from './ProfileCard.constants';
+import {
+  StyledCard,
+  StyledProfileContainer,
+  StyledAvatar,
+  StyledInfoContainer,
+  StyledName,
+  StyledUsername,
+  StyledRole,
+  StyledStatsContainer,
+  StyledStatItem,
+  StyledStatIcon,
+  StyledStatContent,
+  StyledStatValue,
+  StyledStatLabel,
+} from './ProfileCard.styles';
+
+import type { ProfileCardProps, StatsItemProps } from './ProfileCard.types';
+
+const StatsItem: React.FC<StatsItemProps> = ({
+  icon,
+  label,
+  value,
+  $variant,
+  'data-testid': dataTestId,
+}) => {
+  const formattedValue = typeof value === 'number' ? value.toLocaleString('en-US') : value;
```

Comment:
Number formatting is using a hardcoded locale (`toLocaleString('en-US')`) in StatsItem. This prevents correct formatting for other locales.

**Sugerencia:**
```typescript
Use the user's locale (e.g. `Intl.NumberFormat(undefined).format(value)`) or accept a `locale` prop / use a localization context. Avoid hardcoding `en-US`.
```